src/server/routes/health/health.mocha.ts

```diff
@@ -66,19 +69,32 @@ describe("health router", () => {
       server.close(done);
     });
 
+    let logSpy: SinonSpy;
+
+    beforeEach(() => {
+      logSpy = spy(LOGGER, "log");
+    });
+
+    afterEach(() => {
+      logSpy.restore();
+    });
+
     const singleClusterTests = [
-      { scenario: "healthy broker", status: 200, initialized: true, delay: 0, expectedStatus: 200 },
-      { scenario: "unhealthy broker", status: 500, initialized: false, delay: 0, expectedStatus: 503 },
-      { scenario: "uninitialized broker", status: 200, initialized: false, delay: 0, expectedStatus: 503 },
-      { scenario: "timeout to broker", status: 200, initialized: true, delay: 200, expectedStatus: 503 }
+      { scenario: "healthy broker", status: 200, initialized: true, delay: 0, expectedStatus: 200, expectedLoggerCalled: false },
+      { scenario: "unhealthy broker", status: 500, initialized: false, delay: 0, expectedStatus: 503, expectedLoggerCalled: true },
+      { scenario: "uninitialized broker", status: 200, initialized: false, delay: 0, expectedStatus: 503, expectedLoggerCalled: true },
+      { scenario: "timeout to broker", status: 200, initialized: true, delay: 200, expectedStatus: 503, expectedLoggerCalled: true }
     ];
 
-    singleClusterTests.forEach(({ scenario, status, initialized, delay, expectedStatus }) => {
+    singleClusterTests.forEach(({ scenario, status, initialized, delay, expectedStatus, expectedLoggerCalled }) => {
       it(`returns ${expectedStatus} with ${scenario}`, testComplete => {
         mockLoadStatus(wikiBrokerNock, { status, initialized, delay });
         supertest(app)
           .get("/")
-          .expect(expectedStatus, testComplete);
+          .expect(expectedStatus, () => {
+            expect(logSpy.calledOnce).to.be.eq(expectedLoggerCalled);
```

Comment:
Is this idiomatic way to check this spy?

Reply:
Idiomatic or not it is unreadable now
I would totally discard testing if we are really logging in health handler